python/pyspark/ml/connect/functions.py

```diff
@@ -31,3 +31,43 @@ def array_to_vector(col: Column) -> Column:
 
 
 array_to_vector.__doc__ = PyMLFunctions.array_to_vector.__doc__
+
+
+# predict_batch_udf is compatible with Spark Connect,
+# add it here to reuse the doctest.
+def _predict_batch_udf() -> Column:
```

Comment:
@HyukjinKwon I want to test the doctest of `predict_batch_udf` in connect mode, it seems  I have to add a dummy function here. Is there some elegant way to do this?

Reply:
I think we can just have a function `predict_batch_udf` that calls `PyMLFunctions.predict_batch_udf`

Reply:
good idea!